### PR TITLE
Add sizes support to AdoceanBidder

### PR DESCRIPTION
**Pull request by AndriyPavlyuk**

### CODE REVIEW

**Review by snahornyi — DISMISSED** [review]

File: src/main/java/org/prebid/server/bidder/adocean/AdoceanBidder.java [hunk 1]
```diff
@@ -125,13 +132,19 @@ private boolean addRequestAndCheckIfDuplicates(List<HttpRequest<Void>> httpReque
                         continue;
                     }
 
+                    queryParams.add(new BasicNameValuePair("aid", extImpAdocean.getSlaveId() + ":" + impid));
+                    final List<String> sizeValues = test != null && test == 1
+                            ? setSlaveSizesParam(slaveSizes, true)
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/bidder/adocean/AdoceanBidder.java:137]: What the difference?

File: src/main/java/org/prebid/server/bidder/adocean/AdoceanBidder.java [hunk 2]
```diff
@@ -170,38 +207,55 @@ private String buildUrl(String impid, ExtImpAdocean extImpAdocean, String consen
...
-                    request.getDevice().getUa());
-
-            if (StringUtils.isNotBlank(request.getDevice().getIp())) {
-                HttpUtil.addHeaderIfValueIsNotEmpty(headers, HttpUtil.X_FORWARDED_FOR_HEADER,
-                        request.getDevice().getIp());
-            } else if (StringUtils.isNotBlank(request.getDevice().getIpv6())) {
-                HttpUtil.addHeaderIfValueIsNotEmpty(headers, HttpUtil.X_FORWARDED_FOR_HEADER,
-                        request.getDevice().getIpv6());
-            }
+            addHeader(headers, "User-Agent", request.getDevice().getUa());
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/bidder/adocean/AdoceanBidder.java:234]: Check for nulls in values for headers.

**Review by rpanchyk — CHANGES_REQUESTED** [review]

File: src/main/java/org/prebid/server/bidder/adocean/AdoceanBidder.java (hunk 2 above)
Comment (snahornyi) [reply src/main/java/org/prebid/server/bidder/adocean/AdoceanBidder.java:234]: method addHeader contains StringUtils.isNotBlank(value)

Comment (rpanchyk) [reply src/main/java/org/prebid/server/bidder/adocean/AdoceanBidder.java:234]: fair

**Review by rpanchyk — APPROVED** [review]

**Pull request merged**